clippy_lints/src/doc/mod.rs

```diff
@@ -1282,6 +1312,7 @@ fn check_doc<'a, Events: Iterator<Item = (pulldown_cmark::Event<'a>, Range<usize
                 headers.safety |= in_heading && trimmed_text == "Implementation Safety";
                 headers.errors |= in_heading && trimmed_text == "Errors";
                 headers.panics |= in_heading && trimmed_text == "Panics";
+                headers.examples |= in_heading && trimmed_text == "Examples";
```

Comment:
This is starting to becoming a long list of repetition. It would probably better to have something like:

```rust
if in_heading {
    match trimmed_text {
        "Safety" | "SAFETY" | … => headers.safety = true,
        …
    }
}
```